benefits/templates/admin/agency-base.html

```diff
@@ -36,45 +33,46 @@ <h1 class="p-0 m-0 text-white">{{ site_header }}</h1>
             {% block welcome-msg %}
                 <span class="text-uppercase text-white">
                     Welcome,
-                    <span class="fw-bold">{% firstof user.get_short_name user.get_username %}</span>. </span>
-                {% endblock welcome-msg %}
-                {% block userlinks %}
-                    {% if user.is_active and user.is_staff %}
-                        {% url 'django-admindocs-docroot' as docsroot %}
-                        {% if docsroot %}<a href="{{ docsroot }}">Documentation</a> /{% endif %}
-                    {% endif %}
-                    {% if user.has_usable_password %}
-                        <a href="{% url 'admin:password_change' %}">Change password</a> /
-                    {% endif %}
-                    <img class="icon" width="15" height="15" src="{% static "img/icon/box-arrow-right.svg" %}" alt="" />
-                    <form id="logout-form" method="post" action="{% url 'admin:logout' %}">
-                        {% csrf_token %}
-                        <button type="submit" class="border-0">Log out</button>
-                    </form>
-                {% endblock userlinks %}
-            </div>
-        {% endif %}
-    {% endblock usertools %}
+                    <span class="fw-bold">{% firstof user.get_short_name user.get_username %}</span>.
+                </span>
+            {% endblock welcome-msg %}
+            {% block userlinks %}
+                {% if user.is_active and user.is_staff %}
+                    {% url 'django-admindocs-docroot' as docsroot %}
+                    {% if docsroot %}<a href="{{ docsroot }}">Documentation</a> /{% endif %}
+                {% endif %}
+                {% if user.has_usable_password %}
+                    <a href="{% url 'admin:password_change' %}">Change password</a> /
+                {% endif %}
+                <img class="icon" width="15" height="15" src="{% static "img/icon/box-arrow-right.svg" %}" alt="" />
+                <form id="logout-form" method="post" action="{% url 'admin:logout' %}">
+                    {% csrf_token %}
+                    <button type="submit" class="border-0">Log out</button>
+                </form>
+            {% endblock userlinks %}
+        </div>
+    {% endif %}
+{% endblock usertools %}
```

Comment:
I think we should extract this `usertools` block out as an include template as well. Your thoughts @lalver1?

Reply:
Yep, I was also wondering about this @angela-tran. Ah, because `usertools` will be used in a few places in PR #2423  it would be good to move it to an include template 👍

Reply:
Moved `usertools` to an include template in 6ee7e3cc222092ecb9ef21d92f3470768aa851b9